executor/show.go

```diff
@@ -480,7 +480,9 @@ func (e *ShowExec) fetchShowColumns(ctx context.Context) error {
 		if e.Column != nil && e.Column.Name.L != col.Name.L {
 			continue
 		}
-
+		if col.Tp == mysql.TypeVarString {
```

Comment:
why not change the `tp` to `TypeVarhar` when creating a view?

Reply:
In L463 comment, it shows that view's undertable's column could change or recreate. So we can't set tp to `TypeVarchar` when creating a view.
Or, you mean that changing the tp to `TypeVarchar` in L473?

Reply:
Addressed.